### PR TITLE
Test Kitchen Config Refactor

### PR DESCRIPTION
**Pull request by zwall-bp**
All tests green, except for dokken suffering from issues in:
* mgr
* mds
* osd

Anything that utilizes `mds` has issues with spinning up the instance. Logs only provide:
```
Oct 16 22:58:49 node1 ceph-mds[15264]: 2024-10-16 22:58:49.297 7f75f0a70700  1 mds.node1 Updating MDS map to version 265 from mon.0
Oct 16 22:58:52 node1 ceph-mds[15264]: 2024-10-16 22:58:52.589 7f75f0a70700  1 mds.node1 Updating MDS map to version 266 from mon.0
Oct 16 22:58:57 node1 ceph-mds[15264]: 2024-10-16 22:58:57.070 7f75f0a70700  1 mds.node1 Updating MDS map to version 267 from mon.0
Oct 16 22:59:01 node1 ceph-mds[15264]: 2024-10-16 22:59:01.470 7f75f0a70700  1 mds.node1 Updating MDS map to version 268 from mon.0
Oct 16 22:59:09 node1 ceph-mds[15264]: 2024-10-16 22:59:09.530 7f75f0a70700  1 mds.node1 Updating MDS map to version 269 from mon.0
Oct 16 22:59:12 node1 ceph-mds[15264]: 2024-10-16 22:59:12.950 7f75f0a70700  1 mds.node1 Updating MDS map to version 270 from mon.0

```

### CODE REVIEW

File: resources/mon.rb [hunk 1]
```diff
@@ -43,10 +43,17 @@
     creates "/var/lib/ceph/mon/ceph-#{node['hostname']}/done"
   end
 
+  # Docker has extra protections in place which are preventing the execution of the next resource
+  execute 'Disable fs issue' do
+    command 'sysctl -w fs.protected_regular=0'
+    action :nothing
+  end
+
```
Comment (ramereth) [inline resources/mon.rb:51]: Please use the [sysctl resource](https://docs.chef.io/resources/sysctl/) instead.

**Review by ramereth — CHANGES_REQUESTED** [review]

**Comment by ramereth** [comment]
@zwall-bp I'm having a lot of trouble with dokken due to the use of lvm. My workstation also uses lvm and so it gets mixed up with the system lvm.

Let's just remove dokken support from this cookbook.

**Comment by zwall-bp** [comment]
10-4

**Comment by ramereth** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v6.5.3.  Have a nice day!

**Pull request merged**